assets/js/http.js

```diff
@@ -11,4 +11,52 @@ class Http {
 
     return data;
   }
+
+  async post(url, data) {
+    const response = await fetch(url, {
+      method: "POST",
+      headers: {
+        "Content-type": "application/json",
+      },
+      body: JSON.stringify(data),
+    });
+
+    const resData = await response.json();
+
+    return resData;
+  }
+
+  async put(url, data) {
+    const response = await fetch(url, {
+      method: "PUT",
+      headers: {
+        "Content-type": "application/json",
+      },
+      body: JSON.stringify(data),
+    });
+
+    if (!response.ok) {
```

Comment:
Nitpick: Ideally you'd also add error handling to the `post` method as well. 😃   
Although, I'd say that "eating away" the response and only returning a formatted message is not the best approach as you might want to do some custom error handling on the consuming side.  
You could also experiment with just returning a `Promise.reject(response)`, but in any case, these are just comments to encourage you to think about various cases, but in your case it is perfectly fine. 👍